docker-storage-setup.sh

```diff
@@ -878,42 +875,46 @@ get_existing_storage_driver() {
   return 1
 }
 
-docker_root_volume_exists() {
-  lvs $VG/$DOCKER_ROOT_LV_NAME > /dev/null 2>&1 && return 0
+extra_volume_exists() {
+  lvs $VG/$1 > /dev/null 2>&1 && return 0
   return 1
 }
 
-# This returns the mountpoint of $DOCKER_ROOT_LV_NAME
-docker_root_lv_mountpoint() {
+# This returns the mountpoint of $1
+extra_lv_mountpoint() {
   local mounts
-  mounts=$(findmnt -n -o TARGET --first-only --source /dev/$VG/$DOCKER_ROOT_LV_NAME)
+  mounts=$(findmnt -n -o TARGET --first-only --source /dev/$VG/$1)
   echo $mounts
 }
 
-# Mount docker root volume on $DOCKER_ROOT_DIR.
-# drop a systemd mount unit configuration file at /etc/systemd/system.
-# This would allow the $DOCKER_ROOT_DIR mountpoint to persist across reboots.
-mount_docker_root_volume() {
+# This method is used to setup mountfile for both, $docker_root_lv_name
+# and $extra_volume_name. $extra_volume_name will be mounted
+# on $resolved_mount_dir_path. $docker_root_lv_name
+# will be mounted on $docker_root_dir. Drop a systemd mount unit
+# configuration file at /etc/systemd/system.
+# This would allow the mountpoint to persist across reboots.
+mount_extra_volume() {
   local filename
-  # filename must match $DOCKER_ROOT_DIR path. e.g if $DOCKER_ROOT_DIR is /var/lib/docker
-  # then filename will be var-lib-docker.mount
-  filename=$(systemd_escaped_filename ${DOCKER_ROOT_DIR})
+  # filename must match the path ${2}.
+  # e.g if ${2} is /var/lib/containers
+  # then filename will be var-lib-containers.mount
+  filename=$(systemd_escaped_filename ${2})
```

Comment:
same here, lets save pass in parameters in a local variable name and use that variable name in the function.